pkg/agent/openflow/pipeline.go

```diff
@@ -2457,15 +2463,22 @@ func (f *featureService) serviceEndpointGroup(groupID binding.GroupIDType, withS
 		ipProtocol := getIPProtocol(endpointIP)
 
 		if ipProtocol == binding.ProtocolIP {
-			ipVal := binary.BigEndian.Uint32(endpointIP.To4())
-			group = group.Bucket().Weight(100).
-				LoadToRegField(EndpointIPField, ipVal).
-				LoadToRegField(EndpointPortField, uint32(portVal)).
-				ResubmitToTable(resubmitTableID).
-				Done()
+			// Antrea Multi-cluster support IPv4 only for now.
+			if mcsLocalService != nil && endpointIP.Equal(mcsLocalService.ClusterIP) {
```

Comment:
Why we have such scenario that `mcsLocalService != nil` but the endpoints does not contain "mcsLocalService.ClusterIP", and why not just set `mcsLocalService==nil` in that case?

Reply:
I don't get your point here, you mean to install normal Endpoint bucket with the check `mcsLocalService==nil`? This is in a loop of Endpoints list, there will be a case uncovered, we also need to install normal Endpoint bucket if `mcsLocalService != nil` and current Endpoint is not local Service's ClusterIP.

Reply:
My point is we should trust that the provided `mcsLocalService` and `endpoints` are desired to install in the OF Group, and no need to check the duplicated items in pkg/agent/openflow/pipeline.go . The caller of this function should guarantee all the parameters are valid.

Reply:
Done, pass the Endpoint without ClusterIP when mcsLocalService is not nil.